doc/source/diagnostics/slow_extended_55.rst

```diff
@@ -147,10 +149,10 @@ To stop the logging from the slave thread, you should just change the global val
      :scope: Global
      :dyn: Yes
      :vartype: Boolean
-     :default: TRUE
-     :range: TRUE/FALSE
+     :default: ON
+     :range: ON/OFF/OFF_NO_CALLS
 
-If ``TRUE``, statements executed by stored procedures are logged to the slow if it is open.
+This variable controls the logging of slow ``CALL`` statements, logging individual slow statements inside stored procedures or skipping the logging of stored procedures into the slow log entirely. Values are ``OFF``, ``ON`` and ``OFF_NO_CALLS`` respectively.
```

Comment:
Hard to understand the effect of "OFF", maybe something like: "This variable controls the logging of stored procedures in the slow query log. The options are to log the CALL statements, to log the individual statements in the stored procedure, or to log neither. These options correspond to values "OFF", "ON", and "OFF_NO_CALLS" respectively.